### ISSUE TITLE
[Bug] antlrv4 plugin v1.8.3 fails to generate parse tree visitor for spark sql (SqlBase.g4)

### ISSUE THREAD

**Issue by debugger87** [antlr/intellij-plugin-v4#292]
* Intellij Idea Env

```
IntelliJ IDEA 2016.3.1
Build #IU-163.9166.29, built on December 9, 2016
JRE: 1.8.0_112-release-408-b2 x86_64
JVM: OpenJDK 64-Bit Server VM by JetBrains s.r.o
```

* SqlBase.g4

antlrv4 plugin v1.8.2 works well for [SqlBase.g4](https://github.com/apache/spark/blob/master/sql/catalyst/src/main/antlr4/org/apache/spark/sql/catalyst/parser/SqlBase.g4), but v1.8.3 fails.

**Comment by parrt** [comment]
Any hints about error messages etc...?

**Comment by parrt** [comment]
Ah. No input even. I get an exception.

```
antlr can't process SqlBase.g4
java.lang.NullPointerException
	at org.antlr.v4.automata.ParserATNFactory.elemList(ParserATNFactory.java:468)
	at org.antlr.v4.automata.ParserATNFactory.alt(ParserATNFactory.java:455)
	at org.antlr.v4.parse.ATNBuilder.alternative(ATNBuilder.java:559)
	at org.antlr.v4.parse.ATNBuilder.ruleBlock(ATNBuilder.java:293)
	at org.antlr.v4.automata.ParserATNFactory._createATN(ParserATNFactory.java:161)
	at org.antlr.v4.automata.LexerATNFactory.createATN(LexerATNFactory.java:120)
	at org.antlr.v4.Tool.processNonCombinedGrammar(Tool.java:389)
	at org.antlr.v4.Tool.process(Tool.java:354)
	at org.antlr.intellij.plugin.ANTLRv4ExternalAnnotator.doAnnotate(ANTLRv4ExternalAnnotator.java:112)
	at org.antlr.intellij.plugin.ANTLRv4ExternalAnnotator.doAnnotate(ANTLRv4ExternalAnnotator.java:44)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.doAnnotate(ExternalToolPass.java:184)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass.access$500(ExternalToolPass.java:46)
	at com.intellij.codeInsight.daemon.impl.ExternalToolPass$1.run(ExternalToolPass.java:126)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:337)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:327)
	at com.intellij.util.ui.update.MergingUpdateQueue$3.run(MergingUpdateQueue.java:271)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:286)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:244)
	at com.intellij.util.ui.update.MergingUpdateQueue.run(MergingUpdateQueue.java:234)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.Alarm$Request$1.run(Alarm.java:352)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at com.intellij.util.concurrency.QueueProcessor$RunnableConsumer.consume(QueueProcessor.java:298)
	at com.intellij.util.concurrency.QueueProcessor$RunnableConsumer.consume(QueueProcessor.java:295)
	at com.intellij.util.concurrency.QueueProcessor$2$1.run(QueueProcessor.java:110)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.concurrency.QueueProcessor$2.consume(QueueProcessor.java:107)
	at com.intellij.util.concurrency.QueueProcessor$2.consume(QueueProcessor.java:104)
	at com.intellij.util.concurrency.QueueProcessor$3$1.run(QueueProcessor.java:215)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:238)
	at com.intellij.util.concurrency.QueueProcessor$3.run(QueueProcessor.java:212)
	at com.intellij.openapi.application.impl.ApplicationImpl$8.run(ApplicationImpl.java:365)
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:471)
	at java.util.concurrent.FutureTask.run(FutureTask.java:262)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:615)
	at java.lang.Thread.run(Thread.java:745)
	at org.jetbrains.ide.PooledThreadExecutor$1$1.run(PooledThreadExecutor.java:55)
```

**Comment by parrt** [comment]
Rats. this is really an antlr bug.

**Comment by parrt** [comment]
Fixed by pulling in a preview of ANTLR 4.7.1
